.travis.yml

```diff
@@ -20,8 +20,8 @@ before_script:
     - psql -U postgres -c "CREATE USER pulp WITH SUPERUSER LOGIN;"
     - psql -U postgres -c "CREATE DATABASE pulp OWNER pulp;"
 script:
-    # flake8 the diff
-    - "git diff HEAD^ '*.py' | flake8 --diff --config flake8.cfg"
```

Comment:
We originally did it this way because most of pulp failed flake8. I'm happy with this change assuming this passes when merged.